core/market/decentralized/src/rest_api/provider.rs

```diff
@@ -42,11 +43,11 @@ async fn subscribe(
 }
 
 #[actix_web::get("/offers")]
-async fn get_offers(market: Data<Arc<MarketService>>, id: Identity) -> HttpResponse {
-    match market.get_offers(Some(id)).await {
-        Ok(offers) => HttpResponse::Ok().json(offers),
-        Err(error) => HttpResponse::InternalServerError().json(&format!("{}", error)),
-    }
+async fn get_offers(market: Data<Arc<MarketService>>, id: Identity) -> impl Responder {
+    market
+        .get_offers(Some(id), Utc::now().naive_utc())
```

Comment:
All time setting fns (especially `Utc::now().naive_utc()`) are now enclosed withing `SubscriptionStore` and here REST API endpoint is for sure not good place for doing this.